src/build/README.md

```diff
@@ -8,7 +8,7 @@ The Cudos Network is currently in a public testnet entitled 'Somniorum' ([See wh
 
 Developers can start building smart contracts on Somniorum today and be ready to launch their products at launch of Cudos Network mainnet.
 
-[Start a Somniorum Validator](/build/validator.html), or [dive into our docs now to start building!](/build/smart-contracts.html)
+[Start a Somniorum Validator](/build/validator.html), or [dive into our docs now to start building!](/build/smart-contract/developers-setup.html)
```

Comment:
@Nazeim could we ensure that we have no broken links in production please? I've just changed this one in this PR but this came from master, so the docs have the broken link

Reply:
also, please avoid creating structure that does not translate into the docs -- here you have a level called `smart-contract` that does not have any page or reflection in the docs, so it's unnecessary.

That being said, I would separate this doc in three parts, reorganising the whole structure for the devs section, please see my comment at the start of that section